### PR TITLE
zh-cn: translation of mdn/community/learn-forum and related small fixes

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/mdn/community/learn_forum/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+---
+{{MDNSidebar}}
+
+我们的[学习 web 开发](/zh-CN/docs/Learn)页面每个月可以获得上百万次的观看量，且拥有[活跃的论坛](https://discourse.mozilla.org/c/mdn/learn/250)可供人们寻求普遍的帮助，或请求批改他们的作业。我们乐于帮助人们回答帖子中的问题，以让这个学习社区更加壮大。
+
+## 我们需要的帮助
+
+在 [MDN 学习论坛](https://discourse.mozilla.org/c/mdn/learn/250)中，主要需要帮助回答以下两类贴子中的问题：
+
+1. 有关 web 开发的一般问题。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/learn_forum/index.md:13]:
```suggestion
1. 有关 web 开发的常见问题。
```

File: files/zh-cn/mdn/community/learn_forum/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+2. 需要帮助的特定问题，及出现在 MDN web 开发学习区，需要评估或帮助的技能测试和作业题。
+
+## 如何可以受益
```
Comment (yin1999) [inline files/zh-cn/mdn/community/learn_forum/index.md:16]:
```suggestion
## 如何受益
```

File: files/zh-cn/mdn/community/learn_forum/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+- 帮助人们解决他们的代码问题是学习更多网络技术的好方法——当你研究一个解决方案并为别人的问题写出答案时，你会对这个主题有更深的理解，并提高你的技能。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/learn_forum/index.md:18]:
```suggestion
- 帮助人们解决他们的代码问题是学习更多 web 技术的好方法——当你研究一个解决方案并为别人的问题写出答案时，你会对这个主题有更深的理解，并提高你的技能。
```

File: files/zh-cn/mdn/community/learn_forum/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+- 随着你在 MDN 社区的参与度提升，你会认识到更多 Mozilla 的工作人员和其他社区成员，这些人可以增加宝贵的接触机会，帮助你的问题得到解决，提升你的知名度。
+- 帮助回答编码问题在很大程度上是对自己的奖励，但它也将展示你在网络技术方面的专业知识，甚至可能帮助你的课程成绩得到提升，或找到工作机会。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/learn_forum/index.md:20]:
```suggestion
- 帮助回答编码问题在很大程度上是对自己的奖励，但它也将展示你在 web 技术方面的专业知识，甚至可能帮助你的课程成绩得到提升，或找到工作机会。
```

**Review by yin1999 — APPROVED** [review]
Almost good, some suggestions are listed:

File: files/zh-cn/mdn/community/learn_forum/index.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+2. 如果还不熟悉相关内容，请看一下[学习 web 开发](/zh-CN/docs/Learn)部分，并对那里的内容有一些初步的了解（请查阅下述 [MDN 学习区的结构](#mdn_学习区的结构)一节）。
+
+当你准备好了之后：
+
+1. 查看[学习论坛](https://discourse.mozilla.org/c/mdn/learn/250)，留意那些没有回复的贴子——这往往是好的出发点。
+
+   - 提示：如果你找不到没有回复的贴子，检查那些最近更新的，并看看能不能添加一些还没有提及过的有用的内容。
+
+2. 如果你所回复的贴子是一般性的求助，那就回复他们，并在你有时间的情况下给予他们尽可能多的帮助。
+3. 如果你所回复的贴子是请求“技能测试”/“作业” 任务的评估：
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/learn_forum/index.md:39]:
```suggestion
3. 如果你所回复的贴子是请求“技能测试”或“作业”任务的评估：
```

**Pull request merged**